packages/build/bin/compile-package.js

```diff
@@ -110,6 +116,18 @@ function run(argv, options) {
 
   if (outDir) {
     args.push('--outDir', outDir);
+
+    if (rootDir && tsConfigFile && isCopyResourcesSet) {
+      const tsConfig = require(tsConfigFile);
+      if (tsConfig.include) {
```

Comment:
Should be follow `extends` in `tsConfigFile` recursively, merge the results and use that? I'm not sure how we can implement this without knowing the directories to move the resources in. And this is in the `outDir` if-block since we also need to know the outDir. If it's not set (compile in place) then we have no need to move the resources. Hence my choices so far. 

cc: @raymondfeng